packages/airnode-deployer/package.json

```diff
@@ -11,10 +11,10 @@
     "dist"
   ],
   "scripts": {
-    "build": "yarn clean && yarn compile && yarn webpack && yarn copy:webpack && yarn copy:terraform",
-    "clean": "rimraf -rf ./dist ./.webpack *.tgz",
+    "build": "yarn run clean && yarn compile && yarn webpack && yarn copy:webpack && yarn copy:terraform",
```

Comment:
Why add `run` here but not for the others? I don't think it's needed?

Reply:
I removed all `yarn clean` command at some point, but later reverted it via global find replace. Majority of other commands use `yarn run` so that's why. I've reverted it for this change.